### ISSUE TITLE
ErrorBars not working on categorical axis

### ISSUE THREAD

**Issue by philippjfr** [ioam/holoviews#1470]
The bokeh backend generally support categorical axes but it appears that ErrorBars do not appear when the corresponding x-axis is categorical:

```python
%%opts ErrorBars (line_width=2 line_color='darkblue')
%%opts Points    [invert_axes=False] (color='darkred' size=6)

x=['a','b','c','d','e']
points = [(x[i], 5*np.random.rand()) for i in range(5)]
errors = [points[i]+(np.random.rand()/2,) for i in range(5)]

h1 = hv.Points(points)
h2 = hv.ErrorBars(errors)
h2*h1
```

**Comment by bstadlbauer** [comment]
Hi! 

I have just tried this with a recent `holoviews` release (1.13.1) (`bokeh` version 2.2.1) on MacOS and it seems that there are no whiskers plotted: 

![image](https://user-images.githubusercontent.com/11799671/93873833-b0565100-fcd2-11ea-8064-584c8ad7e1bf.png)

**Comment by gioarma** [comment]
I'm reproducing this error and I'm on Mac too, so maybe it's Mac related?

**Comment by jlstevens** [comment]
I'm also on a Mac and I see no issue on holoviews master. It is possible this issue has been fixed since 1.13.1? One way to find out would be if you could confirm whether you can reproduced with current master.

**Comment by gioarma** [comment]
I'm currently using version `1.14.1` and on MacOS 10.15.7
